contrib/babelfishpg_common/src/collation.c

```diff
@@ -604,12 +610,20 @@ find_any_collation(const char *collation_name, bool check_for_server_collation_n
  */
 static int
 translate_collation_utility(const char *collname)
```

Comment:
this code is being called via `lookup_collation`, can you please check the implication on that?

Reply:
looked into it, should not be affected by our changes. We have modified its definition in such a way that it will work same pre and post our changes